rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/NSState.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.resourcemanager.common;
+
+import java.io.Serializable;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.objectweb.proactive.annotation.PublicAPI;
+
+
+/**
+ * @author ActiveEon Team
+ * @since 30/11/2017
+ */
+
+@PublicAPI
+@XmlRootElement
+public class NSState implements Serializable {
+
+    private boolean result;
```

Comment:
better to have this object as an immutable object. 
To do that you need to :  
1 - make all its properties private final
2 - assign values to its properties in the constructor

To make it easier to create this object you can finally create a builder object with readable methods.